tests/profiler/core/test_profiler_config_parser.py

```diff
@@ -122,37 +96,25 @@ def new_time_profiler_config_parser_path(config_folder):
     return os.path.join(config_folder, "new_time_profiler_config_parser.json")
 
 
-def _convert_to_string(item):
-    return '"{0}"'.format(item) if isinstance(item, str) else item
-
-
-def _convert_key_and_value(key, value):
-    return "{0}: {1}, ".format(_convert_to_string(key), _convert_to_string(value))
-
-
 @pytest.mark.parametrize("test_case", detailed_profiling_test_cases)
-def test_detailed_profiling_ranges(detailed_profiler_config_path, test_case):
+def test_detailed_profiling_ranges(profiler_config_path, test_case):
     profiling_parameters, expected_enabled, expected_can_save, expected_values = test_case
     start_step, num_steps, start_time, duration = profiling_parameters
-    detailed_profiler_config = "{"
-    if start_step:
-        detailed_profiler_config += _convert_key_and_value("StartStep", start_step)
-    if num_steps:
-        detailed_profiler_config += _convert_key_and_value("NumSteps", num_steps)
-    if start_time:
-        detailed_profiler_config += _convert_key_and_value("StartTimeInSecSinceEpoch", start_time)
-    if duration:
-        detailed_profiler_config += _convert_key_and_value("DurationInSeconds", duration)
-    detailed_profiler_config += "}"
+    detailed_profiling_config = form_config(
+        StartStep=start_step,
+        NumSteps=num_steps,
+        StartTimeInSecSinceEpoch=start_time,
+        DurationInSeconds=duration,
+    )
 
     full_config = {
         "ProfilingParameters": {
             "ProfilerEnabled": True,
-            "DetailedProfilingConfig": detailed_profiler_config,
+            "DetailedProfilingConfig": detailed_profiling_config,
```

Comment:
Is this a lint change?

Reply:
No, I just did minor renaming to be consistent.